modules/new_serverpod_auth/serverpod_auth_email_account/serverpod_auth_email_account_server/lib/src/business/email_account_secret_hash.dart

```diff
@@ -9,14 +9,16 @@ import 'package:serverpod_auth_email_account_server/serverpod_auth_email_account
 import 'package:serverpod_auth_email_account_server/src/business/email_account_secrets.dart';
 import 'package:serverpod_shared/serverpod_shared.dart';
 
-/// Class for handling password and verification code hashing in the email account module.
+/// Class for handling password and verification code hashing in the email
```

Comment:
(Applied the 80 char limit to the touched files as well. Though would've also been nice to find out how fast 🐇 learns the new rule.)